cmd/bb_storage/main.go

```diff
@@ -85,10 +93,35 @@ func main() {
 	})
 
 	// RPC server.
-	s := grpc.NewServer(
-		grpc.StreamInterceptor(grpc_prometheus.StreamServerInterceptor),
-		grpc.UnaryInterceptor(grpc_prometheus.UnaryServerInterceptor),
-	)
+	var options []grpc.ServerOption
```

Comment:
Is there any chance we can place this chunk of code under `pkg/`? That way we can also use it in the other components (e.g., bb-scheduler).

Reply:
In some kind of `CreateServerOptions(...)` `func`? Sure, where do you think that should live? I've run out of ideas for naming...

Reply:
Exactly. If we don't feel like creating a separate package for that, I'm fine with throwing that in pkg/util.